speedify.py

```diff
@@ -106,24 +105,37 @@ def wrapper(*args, **kwargs):
 # Functions for controlling Speedify State
 
 
-@exception_wrapper("Failed to connect")
-def connect(server=""):
+def connect(
```

Comment:
I don't think we want to completely change the connect function like this. If you want to add an additional function that lets you specify country, city, num and calls connect, that is fine, but I don't think we should remove the existing behavior. As I mentioned above, it should be valid to call this with no argument.

Reply:
It's not clear to me how to proceed on this one. The previous `connect` function:

```
def connect(server=""):
    """
    connect(server="")
    Tell Speedify to connect. Returns serverInformation if success, raises Speedify if unsuccessful.
    See show_servers() for the list of servers available.

    :param server: Server to connect to.
    :type server: str
    :returns:  dict -- :ref:`JSON currentserver <connect>` from speedify.
    """
    args = ["connect"]
    if server != None and server != "":
        pieces = server.split("-")
        for piece in pieces:
            args.append(piece)
        logger.debug("connecting to server = " + server)

    resultjson = _run_speedify_cmd(args)
    return resultjson
```

Connects to a particular server. The CLI arguments did change, slightly, having added a `p2p` call. 

I could not find matching documentation on the CLI for the existing speedify-py `connect` function, so it was updated to match. 

We can roll this back and implement it differently, but it will be confusingly different from the CLI because this `connect` looks for a server via the splitting `-` search, which is not documented. 

Up to you -- thoughts?

Reply:
There haven't been any significant changes to the connect method, so I think these changes should roll back and stay with the existing connect method. If you want to keep your new connect method that takes multiple arguments as a new helper function, that is fine, but we shouldn't change the behavior on the existing connect function.